Tests/HtmlPrettyPrintTests/__Snapshots__/HtmlPrettyPrintTests/testPretty.1.html

```diff
@@ -13,23 +11,17 @@
                   class2
                   class3"
            style="color: red;
-                  background: blue;
-                  padding: rem(2);">
+                 background: blue;
```

Comment:
just seeing this. looks like we should add one more more space to the indentation...